src/modules/Dashboard/features/Managers/Managers.tsx

```diff
@@ -116,52 +107,110 @@ export const Managers = () => {
+            --
+          </Box>
+        );
 
         return {
-          pin: {
-            content: () => <Pin isPinned={isPinned} data-testid="dashboard-managers-pin" />,
+          checkbox: {
+            content: () => (
+              <Checkbox
+                aria-label={`${firstName} ${lastName}`}
+                checked={false}
+                data-testid="dashboard-managers-checkbox"
+                icon={
+                  <Svg
+                    fill={variables.palette.outline_variant2}
+                    height="20"
+                    id="checkbox-empty-outlined"
+                    width="20"
+                  />
+                }
+              />
+            ),
             value: '',
-            onClick: () => handlePinClick(id),
-            width: ManagersColumnsWidth.Pin,
+            width: '8rem',
+          },
+          avatar: {
+            content: () => (
+              <MuiAvatar
+                sx={{
+                  background: variables.palette.primary_container,
+                  width: '3.2rem',
+                  height: '3.2rem',
+                }}
+              >
+                <Avatar
+                  caption={`${firstName[0] ?? '?'}${lastName[0] ?? '?'}`.toLocaleUpperCase()}
+                  uiType={AvatarUiType.Primary}
+                />
+              </MuiAvatar>
+            ),
+            value: '',
+            width: '8rem',
           },
           firstName: {
-            content: () => firstName,
+            content: () => firstName ?? emptyContent,
             value: firstName,
-            width: ManagersColumnsWidth.Default,
           },
           lastName: {
-            content: () => lastName,
+            content: () => lastName ?? emptyContent,
             value: lastName,
-            width: ManagersColumnsWidth.Default,
           },
-          email: {
-            content: () => email,
-            value: email,
-            width: ManagersColumnsWidth.Email,
+          title: {
+            content: () => emptyContent,
+            value: '',
           },
           ...(appletId && {
             roles: {
-              content: () => stringRoles,
+              content: () => <>{renderedRoles ?? emptyContent}</>,
               value: stringRoles,
-              width: ManagersColumnsWidth.Default,
             },
           }),
+          email: {
+            content: () => email ?? emptyContent,
+            value: email,
+          },
           actions: {
             content: () => {
               if (ownerId === id || !filteredManager?.applets?.length) {
                 return;
               }
 
               return (
-                <ActionsMenu
-                  menuItems={getManagerActions(actions, filteredManager)}
-                  data-testid="dashboard-managers-table-actions"
-                />
+                <Box display="flex" justifyContent="flex-end" alignItems="center" width="100%">
```

Comment:
Please consider using StyledComponents for styling instead of inline properties to maintain consistency with the current codebase.